### DISCUSSION TITLE
Adding latitude and longitude frames?

### DISCUSSION THREAD

**Discussion by AndresSepulveda** [NCAR/VAPOR#3582] [Q&A] [answered]
Hi,

I am using VAPOR to animate results from WRF-ARW. Could you tell me how to add the latitude/longitude  tickmarks to the frame?

**Comment by sgpearse** [comment, marked as answer]
Hi @AndresSepulveda - Axis annotations can be found under the Scene->Annotation tab on the left side of the GUI.  Screenshot below:

![Screen Shot 2024-04-15 at 4 54 20 PM](https://github.com/NCAR/VAPOR/assets/9522770/6d5c5e0d-61c3-4d3c-b436-e7f65868ae86)

**Reply by AndresSepulveda** [reply]
Thanks!
